VideoWeb/VideoWeb/ClientApp/src/typings.d.ts

```diff
@@ -39,7 +39,7 @@ declare interface PexipClient {
     onDisconnect: (reason: any) => void;
     onParticipantCreate: (participantUpdate: PexipParticipant) => void;
     onParticipantUpdate: (participantUpdate: PexipParticipant) => void;
-    onParticipantDelete: (participantUpdate: PexipParticipant) => void;
+    onParticipantDelete: (uuid: string) => void;
```

Comment:
To be consistent here, may I suggest `PexipParticipantDeleted`
```suggestion
    onParticipantDelete: (participantUpdate: PexipParticipantDeleted) => void;
```